tests/test_utils/test_utils.py

```
@@ -0,0 +1,629 @@
+    Args:
+        source (str): The path to the source file or directory.
+        target (str): The path to the target symbolic link.
+    """
+    source = os.path.abspath(source)
+    if os.path.exists(target):
+        os.remove(target)
+    os.symlink(source, target)
+
+
+def create_test_package(
+    package_path: str, package_size: str, create_symlinks: bool = True, services: list[str] = []
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)